src/components/Activities/Activity3/Game/GameConstantsToMessWith.ts

```diff
@@ -32,4 +32,10 @@ export const NUMBER_TO_QUALITY_MAP = {
   '2,3': 'not great',
   '1,2': 'poor', // keep a zero value, everything else can be modified
 };
-export const QUALITY_DEFAULT_KEY = '1,2';
\ No newline at end of file
+export const QUALITY_DEFAULT_KEY = '1,2';
+
+//Percent when text no longer fits inside final report bar. Note, this is not a pixel value and may not work well for different sized screens
```

Comment:
nit: if we change to this type of comment, then it will show on hover for `PERCENT_THRESHOLD`
```suggestion
/**
 * Percent when text no longer fits inside final report bar. Note, this is not a pixel value and may not work well for different sized screens
 */
```